calendar-aggregator-boot/pom.xml

```diff
@@ -49,9 +49,30 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>test-jar</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
             <plugin>
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
+                <executions>
```

Comment:
I used the "repackage" goal because of https://docs.spring.io/spring-boot/docs/current/maven-plugin/examples/repackage-classifier.html